CMakeLists.txt

```diff
@@ -1,5 +1,13 @@
 cmake_minimum_required (VERSION 3.5) # The minimum version of CMake necessary to build this project
 
+set( TRISYCL_VERSION_MAJOR 0)
+set( TRISYCL_VERSION_MINOR 1)
+set( TRISYCL_VERSION_PATCH 0)
+set( TRISYCL_VERSION "${TRISYCL_VERSION_MAJOR}.${TRISYCL_VERSION_MINOR}.${TRISYCL_VERSION_PATCH}")
```

Comment:
I guess that the spaces or not inside the '()` should be the same for all the lines of the file?